src/styles/blocks/review.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(../images/star.svg) no-repeat;
+  padding-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
you should keep it operational, what if I will change modifier to 1?
4'th point in checklist